sycl/unittests/get_native_interop/test_get_native.cpp

```diff
@@ -121,11 +121,11 @@ TEST(GetNativeTest, GetNativeHandle) {
   unsigned char *HostAlloc = (unsigned char *)malloc_host(1, Context);
   auto Event = Queue.memset(HostAlloc, 42, 1);
 
-  auto c_g = get_native<backend::opencl>(Context);
-  auto q_g = get_native<backend::opencl>(Queue);
-  auto p_g = get_native<backend::opencl>(Program);
-  auto d_g = get_native<backend::opencl>(Device);
-  auto e_g = get_native<backend::opencl>(Event);
+  get_native<backend::opencl>(Context);
+  get_native<backend::opencl>(Queue);
+  get_native<backend::opencl>(Program);
+  get_native<backend::opencl>(Device);
+  get_native<backend::opencl>(Event);
```

Comment:
Shouldn't this produce a warning?

Reply:
As far as I know, it should not, barring functions with `[[nodiscard]]`.